### PR TITLE
[trivial] routing/router: define ErrRouterShuttingDown

### PR DESCRIPTION
**Pull request by halseth**

### CODE REVIEW

File: routing/router.go [hunk 1]
```diff
@@ -44,6 +44,10 @@ var (
 	// construct a new sphinx packet, but provides an empty set of hops for
 	// each route.
 	ErrNoRouteHopsProvided = fmt.Errorf("empty route hops provided")
+
+	// ErrRouterShuttingDown is returned if the router is in the process of
+	// shutting down.
+	ErrRouterShuttingDown = fmt.Errorf("router shutting down")
```
Comment (halseth) [inline routing/router.go]: Reminder: make part of `routing.Error` type?

Comment (joostjager) [reply routing/router.go]: If we merge this PR, how is this comment going to remind you?

Comment (halseth) [reply routing/router.go]: Don't merge just yet 😛

**Comment by halseth** [comment]
Not ready for merge.

File: routing/router.go (hunk 1 above)
Comment (halseth) [reply routing/router.go]: Did a sweep of the `fundingmgr` and only `routing.ErrOutdated` and `routing.ErrIgnored` is ignored there, so this should be safe to merge.

**Pull request merged**